app/services/web-development.html

```diff
@@ -105,6 +105,33 @@ <h2 class="title my-4">Websites</h2>
           Media setup, Business Emails setup and Search Engine Optimization we create an all-round solution to set the
           ball rolling for your business.
         </p>
+
+        <div class="d-flex justify-content-between align-items-center flex-wrap">
+          <picture>
+            <img src="../images/services/bootstrap-logo.svg" alt="bootstrap logo" title="Bootstrap" class="img-fluid">
+          </picture>
+          <picture>
+            <img src="../images/services/sass.svg" alt="sass logo" title="Sass" class="img-fluid">
+          </picture>
+          <picture>
+            <img src="../images/services/wordpress-logo.svg" alt="wordpress logo" title="Wordpress" class="img-fluid">
```

Comment:
Correct spelling to **WordPress** and capitalization in `alt` and `title`.

Reply:
WordPress typo error is fixed